src/configuration.ts

```diff
@@ -0,0 +1,8 @@
+import { Config } from './types';
+
+export const exampleConfiguration: Config = {
```

Comment:
don't we want to create init() method, that would allow to specify the configuration?

Reply:
Also, it might be an env file with these variables.

Reply:
Example configuration is example of user's configuration. I will remove this after we set the general config.
I will add init method we talked about this.

Reply:
Okay, but remember that we can't store secrets in our public repo. You need to use the .env file anyway.